src/Lykke.AlgoStore.Services/AlgoStoreService.cs

```diff
@@ -96,7 +102,8 @@ public class AlgoStoreService : BaseAlgoStoreService, IAlgoStoreService
                     Margin = instanceData.Margin,
                     HftApiKey = "Dummy HFT Key",
                     HftApiUrl = "Dummy HFT Url",
-                    WalletApiKey = "Dummy Wallet Key"
+                    WalletApiKey = "Dummy Wallet Key",
+                    AlgoInstanceParameters = JsonConvert.SerializeObject(algoInstanceParameters)
                 };
 
                 var response = await _teamCityClient.StartBuild(buildData);
```

Comment:
On line 98 we have AlgoId = data.InstanceId. Shouldn't this be AlgoId = data.AlgoId

Reply:
At the begging of development yes, but at this stage no :)
Eventually, I think that we will completely remove that (and probably other) parameter(s) too and made them part of newly introduced *ALGO_INSTANCE_PARAMS*. This will probably happen in later stages when we are sure that algo is running (probably after demo)